app/renderedMarkdown.tsx

```diff
@@ -1,17 +1,51 @@
 import { marked } from "marked";
 import { RenderedHTML } from "~/renderedHTML";
 
+export function GooeyTooltip(props: {
+  text: string;
+  direction?: "left";
+  iconStyle?: Record<string, string>;
+}) {
+  return (
+    <span
+      className={
+        "gui_tooltip" +
+        (props.direction
+          ? " gui_tooltip--" + props.direction
+          : " gui_tooltip--right")
+      }
+    >
+      <i
+        role="button"
+        className="fa-regular fa-circle-info"
+        style={{ height: "16px", ...props.iconStyle }}
+      ></i>
+      <span className="tooltip_text">
+        <RenderedMarkdown body={props.text} className="" />
```

Comment:
This dependency loop is a bit confusing, can you refactor to have 3 widgets  perhaps, RenderedMarkdownRaw (w/o tooltip), RenderedMarkdown and GooeyTooltip ?